Subject: Add upstream ChangeLog, found by Lintian. Created with svn2cl.
Author: Kai Wasserbäch <debian@carbon-project.org>
Last-Update: 2010-11-25
Forwarded: no
---
 ChangeLog |  477 ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 1 file changed, 477 insertions(+)

--- /dev/null
+++ b/ChangeLog
@@ -0,0 +1,477 @@
+2010-11-30  sars
+
+	* [r1202400] Fix save regression
+	  Fix SaveLocation.cpp/h copyright headers
+
+	  BUG:258370
+
+2010-11-25  sars
+
+	* [r1200802] Add missing find_package(PNG REQUIRED)
+	  Fix save window size bug
+	  Bump version
+
+2010-10-21  dfaure
+
+	* [r1188173] Make executable the Type=Application .desktop files we
+	  install,
+	  so that they can be used even from a KDE running in another prefix
+	  (http://markmail.org/message/4p3krovtnzwx35zd)
+
+2010-09-24  mlaurent
+
+	* [r1179258] Fix compile with gold
+
+2010-08-24  scripty
+
+	* [r1167291] SVN_SILENT made messages (.desktop file)
+
+2010-07-27  scripty
+
+	* [r1155400] SVN_SILENT made messages (.desktop file)
+
+2010-07-09  sars
+
+	* [r1147921] Fix "Remember last saved image path/type" regression.
+
+2010-07-08  sars
+
+	* [r1147579] Save window size on close for next session
+
+2010-07-07  sars
+
+	* [r1147285] Add support for saving 16 bit/color PNG files
+	  Simplify the saving code and add a proper question box for "ask for
+	  first image"
+
+2010-07-05  scripty
+
+	* [r1146075] SVN_SILENT made messages (.desktop file)
+
+2010-06-29  scripty
+
+	* [r1144039] SVN_SILENT made messages (.desktop file)
+	* [r1143913] SVN_SILENT made messages (.desktop file)
+
+2010-06-16  scripty
+
+	* [r1138530] SVN_SILENT made messages (.desktop file)
+
+2010-05-20  scripty
+
+	* [r1128745] SVN_SILENT made messages (.desktop file)
+
+2010-05-11  sars
+
+	* [r1125574] Set a decent preview window size and zoom the image to fit
+	  in that window.
+
+2010-02-15  scripty
+
+	* [r1090501] SVN_SILENT made messages (.desktop file)
+
+2010-01-25  sars
+
+	* [r1080192] Make sure to clean up on close.
+
+	  BUG:190946
+
+	  I'm quite confident that this commit fixes this bug.
+
+2010-01-25  scripty
+
+	* [r1079976] SVN_SILENT made messages (.desktop file)
+
+2010-01-13  sars
+
+	* [r1074306] Revert unneeded fix
+
+	  CCBUG:222542
+
+	  Sorry, I did not check what the original error was, shame on me :)
+
+	  Thanks Pino for fixing the original bug (and for pointing out my
+	  mistake)
+	* [r1074275] Thanks for the patch, patch applied.
+
+	  BUG:222542
+
+2010-01-11  pino
+
+	* [r1073188] simplify constructor call
+
+2010-01-10  scripty
+
+	* [r1072547] SVN_SILENT made messages (.desktop file)
+
+2009-12-28  scripty
+
+	* [r1066910] SVN_SILENT made messages (.desktop file)
+
+2009-11-26  sars
+
+	* [r1054793] Add #ifdefs for libksane features that will be in 4.4 bur
+	  that are not in 4..3
+
+2009-11-18  scripty
+
+	* [r1050931] SVN_SILENT made messages (.desktop file)
+
+2009-10-26  scripty
+
+	* [r1040514] SVN_SILENT made messages (.desktop file)
+
+2009-10-08  aacid
+
+	* [r1032911] remove some executable bits
+
+2009-09-17  scripty
+
+	* [r1024802] SVN_SILENT made messages (.desktop file)
+
+2009-09-08  sars
+
+	* [r1021322] GUI: Reorganize the settings dialog and add the possibility
+	  to specify a preview resolution.
+	  Make the "save preview" image scalable.
+	  Use m_ prefix for class member variables.
+
+2009-08-07  scripty
+
+	* [r1008348] SVN_SILENT made messages (.desktop file)
+
+2009-08-04  scripty
+
+	* [r1006734] SVN_SILENT made messages (.desktop file)
+
+2009-08-03  sars
+
+	* [r1006486] update TODO
+
+2009-06-01  scripty
+
+	* [r976380] SVN_SILENT made messages (.desktop file)
+
+2009-05-09  sars
+
+	* [r965811] Add a "todo" file
+
+2009-05-04  scripty
+
+	* [r963336] SVN_SILENT made messages (.desktop file)
+
+2009-04-29  pino
+
+	* [r961040] set the icon name of a program in the correct way (changing
+	  the internal app name is a bad idea)
+	  CCMAIL: kristof.bal@gmail.com
+
+2009-04-29  kbal
+
+	* [r960960] use the nice 'scanner' icon
+
+2009-04-22  scripty
+
+	* [r957607] SVN_SILENT made messages (.desktop file)
+
+2009-03-23  coles
+
+	* [r943079] Exclamation point squishing.
+
+2009-03-17  sars
+
+	* [r940426] Bump the version number to indicate the new features
+
+2009-02-18  sars
+
+	* [r928077] Add "appending strerror(errno) to some error messages"
+	  (there could be some problems with translation tho...)
+
+	  Thanks Arseniy Lartsev for the patch!
+
+	  BUG: 178457
+	* [r928068] Add a scan mode "Open the save dialog for the first image
+	  only"
+	  Sync the settings on settings dialog exit to ensure that the changes
+	  are saved.
+	  add copyright notice Arseniy Lartsev
+
+	  CCBUG: 177899
+
+2009-01-24  shaforo
+
+	* [r916207] set fine-grained categories for extragear apps.
+	  (in compliance with
+	  http://standards.freedesktop.org/menu-spec/menu-spec-1.0.html#category-registry)
+
+	  as the changes ain't controversial. I'm commiting them w/o
+	  approvement.
+
+	  CCMAIL:tomalbers@kde.nl
+	  CCMAIL:digikam-devel@kde.org
+	  CCMAIL:kile-devel@lists.sourceforge.net
+	  CCMAIL:kare.sars@iki.fi
+	  CCMAIL:jh@kolouch.net
+
+2009-01-21  scripty
+
+	* [r914654] SVN_SILENT made messages (.desktop file)
+
+2009-01-12  scripty
+
+	* [r910011] SVN_SILENT made messages (.desktop file)
+
+2009-01-03  scripty
+
+	* [r905063] SVN_SILENT made messages (.desktop file)
+
+2009-01-02  scripty
+
+	* [r904560] SVN_SILENT made messages (.desktop file)
+
+2008-12-23  scripty
+
+	* [r900712] SVN_SILENT made messages (.desktop file)
+
+2008-12-16  scripty
+
+	* [r897636] SVN_SILENT made messages (.desktop file)
+
+2008-12-12  scripty
+
+	* [r896069] SVN_SILENT made messages (.desktop file, second try)
+
+2008-11-26  scripty
+
+	* [r889325] SVN_SILENT made messages (.desktop file)
+
+2008-11-23  scripty
+
+	* [r888003] SVN_SILENT made messages (.desktop file)
+
+2008-10-26  scripty
+
+	* [r876069] SVN_SILENT made messages (.desktop file)
+
+2008-10-14  scripty
+
+	* [r871270] SVN_SILENT made messages (.desktop file)
+
+2008-09-23  sars
+
+	* [r863892] Fix the license (these files are not part of the libksane
+	  library...)
+	* [r863836] Fix the license (these files are not part of the libksane
+	  library...)
+
+2008-09-20  scripty
+
+	* [r862932] SVN_SILENT made messages (.desktop file)
+
+2008-07-23  sars
+
+	* [r836873] change the version, because we now need the FindKSane.cmake
+	  installed from kdegraphics
+	* [r836872] libksane is not optional for Skanlite -> remove the
+	  optional...
+	  remove the unused cmake dir
+
+2008-07-01  mlaurent
+
+	* [r826722] Remove temporary file
+
+2008-06-06  pino
+
+	* [r817550] remove all the FindKSane copies all around, and optionally
+	  use the FindKSane installed by kdegraphics >= 4.1
+	  adapt to the syntax of that FindKSane
+
+2008-06-06  cgilles
+
+	* [r817534] this is more logic for the moment
+	* [r817515] compile
+
+2008-05-27  mlaurent
+
+	* [r813302] Use INSTALL_TARGETS_DEFAULT_ARGS
+
+2008-05-02  cgilles
+
+	* [r803184] check KSAne from local cmake sub-dir
+
+2008-04-26  mlaurent
+
+	* [r801289] Move cmake file in specific project
+	  it's better when we want to checkout just project
+
+2008-04-25  mlaurent
+
+	* [r800893] Remove not necessary cmake variable
+
+2008-04-13  sars
+
+	* [r796588] Update credits
+	* [r796571] Remove file name from currentDir -> fix auto file name for
+	  the save dialog.
+
+2008-04-09  sars
+
+	* [r795269] Move Skanlite to exrtagear/graphics/
+
+2008-04-09  scripty
+
+	* [r795145] SVN_SILENT made messages (.desktop file)
+	* [r795067] SVN_SILENT made messages (.desktop file, second try)
+
+2008-04-07  scripty
+
+	* [r794381] SVN_SILENT made messages (.desktop file, second try)
+	* [r794320] SVN_SILENT made messages (.desktop file)
+
+2008-04-05  annma
+
+	* [r793954] make Help button work
+
+2008-04-05  sars
+
+	* [r793784] Do not show the settings dialog on startup.
+	  Default save location is now $HOME.
+	  Rearange the settings dialog.
+	* [r793776] Make it possible to build separately
+
+2008-04-05  scripty
+
+	* [r793768] SVN_SILENT made messages (.desktop file)
+
+2008-04-04  scripty
+
+	* [r793714] SVN_SILENT made messages (.desktop file)
+
+2008-04-03  sars
+
+	* [r793319] use toQImage() instead of makeQImage()
+
+2008-04-02  sars
+
+	* [r792988] catalog name not capitalized
+	* [r792986] Set proper window title on the settings dialog and the
+	  scanner selection dialog.
+
+2008-04-02  annma
+
+	* [r792948] make description more useful, add copyright year
+
+2008-04-02  pino
+
+	* [r792945] - Categories is a list, so it must end with the list
+	  separator (';')
+	  - remove unuseful comment
+
+2008-04-02  sars
+
+	* [r792938] Glimpse is dead. Long live Skanlite.
+	* [r792930] rename Glimpse to Skanlite
+
+2008-04-01  scripty
+
+	* [r792449] SVN_SILENT made messages (.desktop file)
+	* [r792426] SVN_SILENT made messages (.desktop file)
+
+2008-03-30  scripty
+
+	* [r791741] SVN_SILENT made messages (.desktop file)
+	* [r791712] SVN_SILENT made messages (.desktop file)
+
+2008-03-27  scripty
+
+	* [r791011] SVN_SILENT made messages (.desktop file)
+	* [r790912] SVN_SILENT made messages (.desktop file)
+
+2008-03-27  annma
+
+	* [r790893] improve doc a bit and fix a few typos
+
+2008-03-25  sars
+
+	* [r790078] rename FindKsane.cmake to FindKSane.cmake and the
+	  corresponding changes to the CMake files.
+	* [r790071] -Add glimpse to the CMakeLists.txt to enable compilation.
+	  -Move the FindKsane.cmake module to cmake/modules/
+	* [r789928] move glimpse to kdereview
+
+2008-03-25  annma
+
+	* [r789800] add a .desktop file. Kare, please can you review the entries
+	  I put there? If in doubt we'll ask people on IRC at some point! The
+	  desktop file allows you to have Glimpse in the K menu and also to find
+	  the doc.
+
+2008-03-19  sars
+
+	* [r787752] Only one connection from the showImageDialog to one of the
+	  save slots
+	  (canceling save produced one extra connect -> save dialog opened twice
+	  the next time)
+
+2008-03-17  sars
+
+	* [r786742] Fix Krazy2 warnings
+
+2008-03-16  mlaurent
+
+	* [r786303] SVN_SILENT:
+	  Minor fix
+
+2008-03-12  sars
+
+	* [r784886] Start using KDialog in stead of QDialog
+
+2008-02-24  sars
+
+	* [r778915] add credit
+	* [r778891] If selectDevice() returns an empty string don't try to open
+	  an empty device.
+	* [r778889] libksane does the insertCatalog() for translations
+
+2008-02-23  sars
+
+	* [r778514] Add a button to display the 'About' dialog.
+	* [r778494] Fix e-mail address
+
+2008-02-03  sars
+
+	* [r770309] Use KDialog in stead of QDialog for the settings
+	  show settings dialog (if not disabled by config) before selecting
+	  scanner
+
+2008-01-30  sars
+
+	* [r768848] Enable translated libksane
+	  Enable setting image quality also for non-autosaved images
+
+	* [r768845] rename pot file to glimpse.pot
+
+2008-01-16  ilic
+
+	* [r762329] Find .ui files too.
+
+2007-12-28  sars
+
+	* [r753939] Change the window title to be the scanner name + Glimpse
+
+2007-12-22  sars
+
+	* [r751542] Add icons to buttons and add i18n to strings (needs to be
+	  done for the .ui too somehow)
+
+2007-12-20  sars
+
+	* [r750951] Q_SLOTS in stead of "slot"
+	* [r750950] -Update About data
+	  -Start using KConfigGroup in stead of QSettings
+
+2007-12-15  sars
+
+	* [r748852] This is the first commit of Glimpse a scanning application
+	  that uses libksane.
+
